### PR TITLE
IGVF-239-a-post-diff-time-unit-update

### PR DESCRIPTION
**Pull request by KAndreeva**
IGVF-239 was redone to resolve conflicts.

### CODE REVIEW

File: src/igvfd/tests/test_schema_differentiated_cell.py [hunk 1]
```diff
@@ -54,3 +54,30 @@ def test_part_of_differentiated_cell(differentiated_cell, differentiated_cell_pa
...
+    assert res.status_code == 422
+
+    res = testapp.patch_json(
+        differentiated_cell['@id'],
+        {'post_differentiation_time_units': 'hour'}, expect_errors=True)
+    assert res.status_code == 422
+
+    res = testapp.patch_json(
+        differentiated_cell['@id'],
+        {'post_differentiation_time': 10, 'post_differentiation_time_units': 'hour'}, expect_errors=True)
```
Comment (cricketsloan) [inline src/igvfd/tests/test_schema_differentiated_cell.py:72]: expect_errors=True, but i don't think you expect errors here

**Review by cricketsloan — APPROVED** [review]
I think in general, when we are not expected in errors we do not put expect_errors=True, but I don't think it breaks if you do.

File: src/igvfd/tests/test_schema_differentiated_cell.py (hunk 1 above)
Comment (KAndreeva) [reply src/igvfd/tests/test_schema_differentiated_cell.py:72]: eliminated 'expect_errors=True'

**Comment by KAndreeva** [comment]
Closing this branch and will have new branch for the rebase.

**Pull request closed without merging**